### PR TITLE
docs(guides) add advanced entry usage example with different filetypes

### PR DESCRIPTION
**Pull request by EugeneHlushko**
- introduce a guide on advanced usage of entry

### CODE REVIEW

File: src/content/guides/entry-advanced.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+console.log('accout page type');
+```
+
+__account.scss__
+
+```scss
+// accout page individual styles
+```
+
+We will use [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/) in 'production' mode for css as a best practice.
```
Comment (montogeek) [inline src/content/guides/entry-advanced.md]:
```suggestion
We will use [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/) in `production` mode for css as a best practice.
```

File: src/content/guides/entry-advanced.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title: Advanced entry
+sort: 24
+contributors:
+  - EugeneHlushko
+---
+
+## Multiple file types per entry
+
+It is possible to provide different types of files when using arrays for [entry](/configuration/entry-context/#entry) value(s) to achieve separate bundles for CSS and JavaScript (and other) files in applicataions that are not using `import` for styles in JavaScript (pre Single Page Applications or different reasons).
```
Comment (montogeek) [inline src/content/guides/entry-advanced.md]:
```suggestion
It is possible to provide different types of files when using an array of values for [entry](/configuration/entry-context/#entry) to achieve separate bundles for CSS and JavaScript (and other) files in applicataions that are not using `import` for styles in JavaScript (pre Single Page Applications or different reasons).
```

Comment (montogeek) [inline src/content/guides/entry-advanced.md]:
```suggestion
It is possible to provide different types of files when using arrays for [entry](/configuration/entry-context/#entry) value(s) to achieve separate bundles for CSS and JavaScript (and other) files in applications that are not using `import` for styles in JavaScript (pre Single Page Applications or different reasons).
```

File: src/content/guides/entry-advanced.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+__account.js__
+
+```javascript
+console.log('accout page type');
+```
+
+__account.scss__
+
+```scss
+// accout page individual styles
```
Comment (montogeek) [inline src/content/guides/entry-advanced.md]:
```suggestion
// account page individual styles
```

File: src/content/guides/entry-advanced.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+__home.scss__
+
+```scss
+// home page individual styles
+```
+
+__account.js__
+
+```javascript
+console.log('accout page type');
```
Comment (montogeek) [inline src/content/guides/entry-advanced.md]:
```suggestion
console.log('account page type');
```

File: src/content/guides/entry-advanced.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+title: Advanced entry
+sort: 24
+contributors:
+  - EugeneHlushko
+---
+
+## Multiple file types per entry
+
+It is possible to provide different types of files when using an array of values for [entry](/configuration/entry-context/#entry) to achieve separate bundles for CSS and JavaScript (and other) files in applicataions that are not using `import` for styles in JavaScript (pre Single Page Applications or different reasons).
```
Comment (EugeneHlushko) [inline src/content/guides/entry-advanced.md]:
```suggestion
It is possible to provide different types of files when using an array of values for [entry](/configuration/entry-context/#entry) to achieve separate bundles for CSS and JavaScript (and other) files in applications that are not using `import` for styles in JavaScript (pre Single Page Applications or different reasons).
```

**Comment by EugeneHlushko** [comment]
@montogeek all valid, applied

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**